packages/monocdk-experiment/build.sh

```diff
@@ -34,5 +34,5 @@ mkdir -p dist/js
 cp ${tarball} dist/js
 
 # copying src/ so this module will also work as a local dependency (e.g. for modules under @monocdk-experiment/*).
-rm -fr src
-rsync -av ${unpacked}/src/ src/
+rm -fr staging
+rsync -av ${unpacked}/ staging/
```

Comment:
Might be faster / avoids wearing SSDs over too much (but could literally also not work, I haven't tried):

```suggestion
# copying staging/ so this module will also work as a local dependency (e.g. for modules under @monocdk-experiment/*
rsync -av --delete ${unpacked}/ staging/
```

Reply:
Now i am realizing I can just move the staging directory from tmp